code/game/machinery/ARES/ARES_interface_apollo.dm

```diff
@@ -382,13 +404,32 @@
 			if(!istype(access_ticket))
 				return FALSE
 			if(access_ticket.ticket_assignee != last_login && access_ticket.ticket_assignee) //must be claimed by you or unclaimed.)
-				to_chat(usr, SPAN_WARNING("You cannot update a ticket that is not assigned to you!"))
+				to_chat(operator, SPAN_WARNING("You cannot update a ticket that is not assigned to you!"))
 				return FALSE
 			access_ticket.ticket_status = TICKET_REJECTED
-			to_chat(usr, SPAN_NOTICE("[access_ticket.ticket_type] [access_ticket.ticket_id] marked as rejected."))
+			to_chat(operator, SPAN_NOTICE("[access_ticket.ticket_type] [access_ticket.ticket_id] marked as rejected."))
 			ares_apollo_talk("Access Ticket [access_ticket.ticket_id]: [access_ticket.ticket_submitter] was rejected access by [last_login].")
 			return TRUE
 
+		if("trigger_vent")
+			playsound = FALSE
+			var/obj/structure/pipes/vents/pump/no_boom/gas/sec_vent = locate(params["vent"])
+			if(!istype(sec_vent) || sec_vent.welded)
+				to_chat(operator, SPAN_WARNING("ERROR: Gas release failure."))
+				playsound(src, 'sound/machines/buzz-two.ogg', 15, 1)
+				return FALSE
+			if(!COOLDOWN_FINISHED(sec_vent, vent_trigger_cooldown))
+				to_chat(operator, SPAN_WARNING("ERROR: Insufficient gas reserve for this vent."))
+				playsound(src, 'sound/machines/buzz-two.ogg', 15, 1)
+				return FALSE
+			to_chat(operator, SPAN_WARNING("Initiating gas release from [sec_vent.vent_tag]."))
+			playsound(src, 'sound/machines/chime.ogg', 15, 1)
+			COOLDOWN_START(sec_vent, vent_trigger_cooldown, COOLDOWN_ARES_VENT)
+			ares_apollo_talk("Nerve Gas release imminent from [sec_vent.vent_tag].")
+			log_ares_security("Nerve Gas Release", "[last_login] released Nerve Gas from Vent '[sec_vent.vent_tag]'.")
+			sec_vent.create_gas(VENT_GAS_CN20_XENO, 6, 5 SECONDS)
```

Comment:
You may want to revisit how either this gas works, or the duration. As is its a bit weird that it is flipping mobs between upright and down while it is pumped, and only then when its done pumping the gas do they remain unconscious for a while.

Reply:
I don't think I can change that. It shouldn't make a difference between being there and not being there on keeping people unconscious, it's likely due to the built up oxy damage rather than a direct effect of the gas. I'll try a tweak and see if it makes a difference.